src/i18n/ja-JP/common.yml

```diff
@@ -0,0 +1,986 @@
+      t: 1 to 2 years
+      t: 1年から2年
+    - key: options.years_of_experience.range_2_5
+      t: 2 to 5 years
+      t: 2年から5年
+    - key: options.years
+    - key: options.years_of_experience.range_5_10
+      t: 5 to 10 years
+      t: 5年から10年
+    - key: options.years_of_experience.range_10_20
+      t: 10 to 20 years
+      t: 10年から20年
+    - key: options.years_of_experience.range_more_than_20
+      t: More than 20 years
+      t: 20年以上
+
+    # years of experience (short versions)
+    - key: options.years_of_experience.range_less_than_1.short
+      t: '<1'
+    - key: options.years_of_experience.range_1_2.short
+      t: 1-2
+    - key: options.years_of_experience.range_2_5.short
+      t: 2-5
+    - key: options.years_of_experience.range_5_10.short
+      t: 5-10
+    - key: options.years_of_experience.range_10_20.short
+      t: 10-20
+    - key: options.years_of_experience.range_more_than_20.short
+      t: '>20'
+
+    # company size
+    - key: options.company_size.range_1
+      t: One employee
+      t: 1人
+    - key: options.company_size.range_1_5
+      t: 1 to 5 employees
+      t: 1人から5人
+    - key: options.company_size.range_5_10
+      t: 5 to 10 employees
+      t: 5人から10人
+    - key: options.company_size.range_10_20
+      t: 10 to 20 employees
+      t: 10人から20人
+    - key: options.company_size.range_20_50
+      t: 20 to 50 employees
+      t: 20人から50人
+    - key: options.company_size.range_50_100
+      t: 50 to 100 employees
+      t: 50人から100人
+    - key: options.company_size.range_100_1000
+      t: 100 to 1000 employees
+      t: 100人から1000人
+    - key: options.company_size.range_more_than_1000
+      t: More than 1000 employees
+      t: 1000人より多い
+
+    # company size (short versions)
+    - key: options.company_size.range_1.short
+      t: 1
+    - key: options.company_size.range_1_5.short
+      t: 1-5
+    - key: options.company_size.range_5_10.short
+      t: 5-10
+    - key: options.company_size.range_10_20.short
+      t: 10-20
+    - key: options.company_size.range_20_50.short
+      t: 20-50
+    - key: options.company_size.range_50_100.short
+      t: 50-100
+    - key: options.company_size.range_100_1000.short
+      t: 100-1000
+    - key: options.company_size.range_more_than_1000.short
+      t: '>1000'
+
+    # salary
+    - key: options.yearly_salary.range_work_for_free
+      t: I work for free
+      t: 無報酬
+    - key: options.yearly_salary.range_0_10
+      t: $0k-$10k
+      t: $0～$10,000
+    - key: options.yearly_salary.range_10_30
+      t: $10k-$30k
+      t: $10,000～$30,000
+    - key: options.yearly_salary.range_30_50
+      t: $30k-$50k
+      t: $30,000～$50,000
+    - key: options.yearly_salary.range_50_100
+      t: $50k-$100k
+      t: $50,000～$100,000
+    - key: options.yearly_salary.range_100_200
+      t: $100k-$200k
+      t: $100,000～$200,000
+    - key: options.yearly_salary.range_more_than_200
+      t: More than $200k
+      t: $200,000以上
```

Comment:
here I convert "_n_k" into numerals as many Japanese are not familiar with the k notation that much.